sklearn/pipeline.py

```diff
@@ -924,12 +924,13 @@ def fit_transform(self, X, y=None, **fit_params):
         X : iterable or array-like, depending on transformers
             Input data to be transformed.
 
-        y : array-like, shape (n_samples, ...), optional
+        y : array-like of shape (n_samples, n_outputs)
```

Comment:
```suggestion
        y : array-like of shape (n_samples, n_outputs), default=None
```

Reply:
@glemaitre Oh, my bad. Thank you for pointing this out.